src/banner/banner.service.ts

```diff
@@ -85,14 +76,15 @@ export class BannerService implements OnDestroy {
 	 * @param {any} [bannerComp=null] If provided, used to resolve component factory
 	 * @memberof BannerService
 	 */
-	showBanner(bannerObj: BannerContent | NotificationContent | ToastContent, bannerComp = null): Banner {
+	showBanner(bannerObj: BannerContent | NotificationContent | ToastContent, bannerComp = null) {
+		let componentFactory;
 		if (!bannerComp) {
```

Comment:
This would read better as "if there's a banner component, use it, otherwise use something else" instead of the negated version we have here.

And "useful" words ;)

Reply:
I realized we could actually just pass a default value of `Banner` and skip the ifs ... so that's coming up.